docs/meetings/client/2022_10_14.md

```diff
@@ -0,0 +1,83 @@
+# Client Meeting Minutes
+
+- **Start Time:** 4:00 PM
+- **End Time:** 5:00 PM
+- **Location:** Zoom
+- **Date:** October 14, 2022
+
+## Attendance
+
+- Client:
+  - Nazim Ragimov (CEO)
+  - Erik (developer)
+
+- Team A:
+  - Edouard Eltherington
+  - Veronica Jack
+  - Logan Parker
+
+***Attendence Comments***
+
+- Matt Kuelker is writing a midterm, so he was unable to attend
+- Students from other teams present as well
+  - Team B only had 1 group member present
```

Comment:
Team C was missing 1 member

Reply:
will add as a note in a re-commit, thanks for catching that!